pkg/operator/clustermembercontroller/clustermembercontroller.go

```diff
@@ -368,9 +368,9 @@ func (c *ClusterMemberController) isEtcdContainerRunningNotReady(node *corev1.No
 	return true, nil
 }
 
-// allNodesMapToVotingMembers returns nodes that don't map to voting members in the etcd cluster membership.
-// The voting members are read from the etcd-endpoints configmap
-func (c *ClusterMemberController) allNodesMapToVotingMembers(nodes []*corev1.Node) ([]*corev1.Node, error) {
+// allNodesMapToNonVotingMembers returns nodes that don't map to voting members (i.e. non learner) in the etcd cluster membership.
+// The voting members are read from the etcd cluster membership.
+func (c *ClusterMemberController) allNodesMapToNonVotingMembers(nodes []*corev1.Node) ([]*corev1.Node, error) {
```

Comment:
@hasbro17  I think this name reflects more what the function does, isn't it ? 

cc @tjungblu